package-lock.json

```diff
@@ -0,0 +1,1199 @@
+{
+  "name": "canvas",
```

Comment:
Delete package-lock.json file

Reply:
Hi @nmgokhale I have deleted package-lock.json file and updated the PR.